### PR TITLE
#160451487 integrate travisCI for continuous integration 

### PR DESCRIPTION
**Pull request by slimsolz**
#### What does this PR do?
Add and configure travis for continuous integration

#### Description of Task to be completed?
1. Add travis.yml file
2. Configure travis to work with node
3. Update readme file with travis badge

#### What are the relevant pivotal tracker stories?
#160451487

### CODE REVIEW

**Comment by samdiano** [comment]
Hi @slimsolz 
The Travis build is failing. I think you should write a simple test to run with so we can get Travis to build successfully.

**Comment by slimsolz** [comment]
@samdiano This task is to setup TravisCI and to add the badge to the readme file, there's a task for adding a test (#160451493) that's why I didn't do that but I will confirm from TTL if I can add a test so it can pass.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+language: node_js
+
+node_js:
+  - "stable"
```
Comment (s-akinrele) [inline .travis.yml]: add an new line

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (s-akinrele) [inline .gitignore]: add a new line

**Pull request merged**